source/MaterialXTest/CMakeLists.txt

```diff
@@ -144,3 +140,8 @@ set_target_properties(
 target_link_libraries(
     MaterialXTest
     ${CMAKE_DL_LIBS})
+
+install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/../../resources"
+    DESTINATION ${CMAKE_INSTALL_PREFIX} )
+install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/../../resources"
+	DESTINATION ${CMAKE_CURRENT_BINARY_DIR} )
```

Comment:
Is this also debug code, or should we copy the resources into the binary directory?

Reply:
There isn't any change in what get's copied. i.e. We're still copying resources to the install directory and the test directory.
This is just cleanup to remove the add_custom_command() which does the same copy.